code-of-conduct-proposed.md

```diff
@@ -52,23 +54,26 @@ Examples of behavior that contributes to a positive environment include:
   and learning from the experience
 * Focusing on what is best not just for us as individuals, but for the
   overall community
+* Using welcoming and inclusive language
+  
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behavior include but are not limited to:
 
 * The use of sexualized language or imagery, and sexual attention or
   advances of any kind
 * Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
+* Public or private harassment in any form
+* Comments or jokes based upon stereotypes, that are exclusionary, or that hold others up for ridicule
```

Comment:
Although I appreciate calling out stereotyping, I think this is probably already be covered by "insulting or derogatory comments" above and thus am not sure if this change is really necessary.  Also, I'm concerned that the new language might cover behavior that we don't want to classify as a CoC violation (e.g., having a conversation about stereotypes) and that not everyone understands what "exclusionary" would mean in this context. Thus, although I think the concerns underlying this change are valid concerns, it might be better to rely on "insulting or derogatory comments" than add this additional language

Reply:
@marinapistarini, please let me know your thoughts on my comment above. The co-chairs met today and once the comments are resolved, they'll approve merging this PR

Reply:
@joannalee333 I agree. I will revert these changes